.env.example

```
@@ -1,20 +1,18 @@
+# API Keys
+OPENAI_API_KEY=your_openai_api_key_here
+ZHIPU_API_KEY=your_zhipu_api_key_here
+ALIYUN_ACCESS_KEY_ID=your_aliyun_access_key_id_here
+ALIYUN_ACCESS_KEY_SECRET=your_aliyun_access_key_secret_here
 
-# Monitoring & Logs
-LOG_LEVEL=INFO
\ No newline at end of file
+# Service Configuration
+PORT=8000
+HOST=0.0.0.0
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The HOST key should go before the PORT key

(UnorderedKey)